hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(string)
+  middle = string.length / 2 # your code here
+
+  if string.length.odd?
+    string[middle]
+  else
+    string[middle - 1] + string[middle]
+  end
+end
```

Comment:
JFYI:
You can change that to ternary operator, like this (one line and do the same):
`string.length.odd? ? string[middle] : string[middle - 1] + string[middle]`

Also, you can use range args and `string[middle - 1] + string[middle]` become `string[(middle - 1)..middle]`